src/Grid/mixed_grid.jl

```diff
@@ -0,0 +1,51 @@
+mutable struct MixedGrid{dim, C, T<:Real} <: Ferrite.AbstractGrid{dim}
+    cells::C # Tuple of concretely typed cell vectors
+    ncells_per_vector::Vector{Int}
+    nodes::Vector{Node{dim,T}}
+    # Sets
+    cellsets::Dict{String,Set{Int}}
+    nodesets::Dict{String,Set{Int}}
+    facesets::Dict{String,Set{FaceIndex}} 
+    edgesets::Dict{String,Set{EdgeIndex}} 
+    vertexsets::Dict{String,Set{VertexIndex}} 
+    # Boundary matrix (faces per cell × cell)
+    boundary_matrix::SparseMatrixCSC{Bool,Int}
+end
+
+function MixedGrid(cells::C,
+        nodes::Vector{Node{dim,T}};
+        cellsets::Dict{String,Set{Int}}=Dict{String,Set{Int}}(),
```

Comment:
I guess `cellsets` should not be a vector of Ints in the MixedGrid?

Reply:
No, working on that :)